downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/DefaultDownloadsRepository.kt

```diff
@@ -57,9 +56,6 @@ class DefaultDownloadsRepository @Inject constructor(
     }
 
     override suspend fun delete(downloadId: Long) {
-        downloadsDatabase.downloadsDao().getDownloadItem(downloadId)?.let {
```

Comment:
This was added when we removed the DownloadManager. File deletion is already handled in the viewmodel.